dotcom-rendering/src/layouts/TagPageLayout.tsx

```diff
@@ -84,7 +84,12 @@ export const TagPageLayout = ({ tagPage, NAV }: Props) => {
 								padSides={false}
 								shouldCenter={false}
 							>
-								<HeaderAdSlot />
+								<HeaderAdSlot
+									isPaidContent={
+										!!tagPage.config.isPaidContent
+									}
+									shouldHideReaderRevenue={false}
```

Comment:
Flagging this here as it appears it's not possible to obtain a `shouldHideReaderRevenue` flag from a tag page layout.